Jenkinsfile

```diff
@@ -82,6 +82,24 @@ pipeline {
         ])
       }
     }
+
+    stage('Rename jenkins workspace path') {
+      agent {label 'nixos-mayastor'}
+      steps{
+        sh """
+        printenv
+        if [[ "${env.WORKSPACE}" =~ "@" ]];
+        then
+            newpath=\$(echo ${env.WORKSPACE} | sed s/"@"/"__"/)
+            mv "${env.WORKSPACE}" $newpath
```

Comment:
Seems a bit "dangerous" to do this, might override something?
Isn't it better to change container build itself (in this repo) to support "@" ?

Reply:
yes , if that can be handled in build code nothing better. So should I close this PR then ?
Also, any idea who can fix it from a build code level. I will create a JIRA and assign to them ?

Reply:
It can also be you if you like nix :) otherwise maybe @niladrih would like to take it?